store/models.py

```diff
@@ -9,6 +9,7 @@ class Book(models.Model):
     description = models.TextField(null=True)
     mrp = models.PositiveIntegerField()
     rating = models.FloatField(default=0.0)
+    user_ratings = JSONField()
```

Comment:
Nice use of JSONField()!
Since the model fields `rating` and `user_rating` are linked with each other, you could have used a [@property decorator,](https://docs.djangoproject.com/en/3.0/topics/db/models/#model-methods) so that the `rating` field would even update if the `user_ratings` is updated from the backend.